test/ci/README.md

```diff
@@ -1,14 +1,6 @@
-This directory contains scripts and other files that are executed by our
-CI integration tests.
+* Copy `test/ci/vars.yml.sample` to `test/ci/inventory/vars.yml`
+* Adjust it your liking
+* Provision instances via `ansible-playbook -vv -i test/ci/inventory/ test/ci/launch.yml -e @test/ci/inventory/vars.yml`
```

Comment:
Let's not use extra_vars if we can help it.

Reply:
Done, rewrote this to use `include_vars` instead